riak/codecs/ttb.py

```diff
@@ -153,6 +153,12 @@ def decode_timeseries(self, resp_ttb, tsobj,
 
         self.maybe_err_ttb(resp_ttb)
 
+        # NB: some queries return a BARE 'tsqueryresp' atom
+        # catch that here:
+        if resp_ttb == tsqueryresp_a:
```

Comment:
@alexmoore - we should make sure the Java client catches this. Some query responses are *not* a tuple but just the atom `tsqueryresp`